### ISSUE TITLE
Set excludeAllHandledTypesClasses as true for servlet-5.0 & Remove Jakarta Prefix Check for HandleTypes

### ISSUE THREAD

**Issue by volosied** [OpenLiberty/open-liberty#17461]
My comment from slack: 
____________
The reason the[ jakarta check was added the WC code ](https://github.com/OpenLiberty/open-liberty/pull/12575)was cause websockets feature failed to initialize properly. The WebSocketServletContainerInitializer  class uses the following annotation
@HandlesTypes({ ServerEndpoint.class, Endpoint.class, ServerApplicationConfig.class }
In EE9, they would be added to the start up class set and passed to WebSocketServletContainerInitializer.onStartUp()  ([javadoc](https://javaee.github.io/javaee-spec/javadocs/javax/servlet/ServletContainerInitializer.html#onStartup-java.util.Set-javax.servlet.ServletContext-))
Since some are interfaces, there would be instantiation errors.
The workaround was to add the jakarta check, similar to the java & javax checks as with pre-EE9 changes.  This was under the assumption jakarta is a reserved package as javax was.  However, this is not the case.
Bill Lucy, however, added a recent [fix](https://github.com/OpenLiberty/open-liberty/pull/16759), where interfaces are skipped entirely with a new property excludeAllHandledTypesClasses. See PR 16759
The property is false (to preserve behavior) for servlet 4.0  & below and true for servlet 5.0+. However, I discovered that it’s not true for servlet-5.0 ( need to remove default value from the metatype.xml to get it working).
With excludeAllHandledTypesClasses then true for servlet-5.0, I think we can undo my change and won’t need Andy’s PR either
____________

In a nutshell,
- Change excludeAllHandledTypesClasses to true or Servlet 5.0
- Undo skipping jakarta packages that are gathered for ServletContainerInitializer#onStartUp 

Original PR: https://github.com/OpenLiberty/open-liberty/pull/12575

Bill's PR: https://github.com/OpenLiberty/open-liberty/pull/16759

Andy's PR:  https://github.com/OpenLiberty/open-liberty/pull/17255
